HW_7.1(Python_data_types).py

```diff
@@ -0,0 +1,65 @@
+# Используя таймер мотоцикла, рассчитайте текущее время. Возвращает ответ в виде суммы цифр,
+# которую покажет цифровой таймер в формате чч:мм.
+# Пример:
+# Для n = 240, результат должен быть: 4
+# 240 минут прошло, текущее время 04:00. Сумма чисел: 0 + 4 + 0 + 0 = 4, это и есть ответ.
+# Для n = 808, результат должен быть: 14
+# 808 минут прошло, и это означает что сейчас 13:28, так что ответ должен быть 1 + 3 + 2 + 8 = 14.
+
+n = 240
+hours = (n // 60) % 24
+minutes = n % 60
+digit_sum = sum(int(digit) for digit in f"{hours:00}{minutes:00}")
+print(digit_sum)
+
+n = 808
+hours = (n // 60) % 24
+minutes = n % 60
+digit_sum = sum(int(digit) for digit in f"{hours:00}{minutes:00}")
+print(digit_sum)
```

Comment:
Code duplication:
```suggestion
n = 240
hours = (n // 60) % 24
minutes = n % 60
digit_sum = sum(int(digit) for digit in f"{hours:00}{minutes:00}")
print(digit_sum)

n = 808
hours = (n // 60) % 24
minutes = n % 60
digit_sum = sum(int(digit) for digit in f"{hours:00}{minutes:00}")
print(digit_sum)
```

What if there are 10, 50, 100 such data?